google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/CloudSdkAppEngineHelper.java

```diff
@@ -186,9 +187,9 @@ public Project getProject() {
         }
         return Optional.of(createFlexRunner(loggingHandler, Paths.get(source.getFilePath()),
             deploymentConfiguration, deploy));
-      } catch (ScannerException se) {
+      } catch (MalformedYamlFile myf) {
```

Comment:
You should add the suffix 'Exception' to the name. I'm pretty sure this is a universal convention.